<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_operate</class>
 <widget class="QDialog" name="D_operate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>D_operate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#D_operate {
    border-image: url(:/bg/resource/bg_most_2.png);
}

QLineEdit {
    background-color: white;
    border: 2px solid white;
    border-radius: 5px;
}

QLineEdit:hover {
    background-color: #F5EFEF;
    border: 2px solid #F5EFEF;
    border-radius: 5px;
}

QLabel {
    color: white;
    font-family: &quot;黑体&quot;;
}

#pushButton_operate_AboutMe{
    border-image: url(:/icons/resource/me_h.png);
}

#pushButton_operate_AboutMe:hover{
    border-image: url(:/icons/resource/me.png);
}


#pushButton_operate_search{
    border-image: url(:/icons/resource/search.png);
}

#pushButton_operate_search:hover{
    border-image: url(:/icons/resource/search_h.png);
}

#pushButton_operate_reset{
    border-image: url(:/icons/resource/reset.png);
}

#pushButton_operate_reset:hover{
    border-image: url(:/icons/resource/reset_h.png);
}

QRadioButton {
    color: white;
}

#pushButton_operate_Prev{
    border-image: url(:/icons/resource/Arrow_Left_Default.png);
}

#pushButton_operate_Prev:hover{
    border-image: url(:/icons/resource/Arrow_Left_Hover.png);
}

#pushButton_operate_Next{
    border-image: url(:/icons/resource/Arrow_Right_Default.png);
}

#pushButton_operate_Next:hover{
    border-image: url(:/icons/resource/Arrow_Right_Hover.png);
}

#pushButton_operate_ask{
    border-image: url(:/icons/resource/pen.png);
}

#pushButton_operate_ask:hover{
    border-image: url(:/icons/resource/pen_h.png);
}

#listWidget_operate_List{
    border: none;
    background: #F5F1F1
}</string>
  </property>
  <widget class="QListWidget" name="listWidget_operate_List">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>183</y>
     <width>455</width>
     <height>477</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_operate_Prev">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>665</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_operate_ask">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>665</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_operate_Next">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>665</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_operate_search_str">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>103</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_operate_search">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>97</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_operate_Title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>136</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>按标题</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_operate_Time">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>136</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>按时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_operate_AboutMe">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>10</y>
     <width>55</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_operate_Hello">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>你好</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_operate_User">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_operate_reset">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>97</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
